roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -32,6 +32,17 @@
   with_items: "{{object_def_files.files | map(attribute='path') | list | sort}}"
   changed_when: no
 
+- oc_obj:
+    state: absent
+    name: "{{item.name}}"
+    kind: "{{item.kind}}"
+    kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
+    namespace: "{{openshift_logging_namespace}}"
+  with_items:
+  - {name: logging-kibana, kind: route}
+  - {name: logging-kibana-ops, kind: route}
+  - {name: kibana-proxy, kind: oauthclient}
```

Comment:
@jcantrill I'm good with deleting and re-creating, I just want to make sure that we are maintaining idempotency and only deleting/recreating when necessary.